docusaurus.config.js

```diff
@@ -13,15 +13,18 @@ module.exports = {
   projectName: 'gulpjs.github.io',
   // The theme
   themeConfig: {
-    disableDarkMode: true,
+    colorMode: {
+      defaultMode: 'light',
+      disableSwitch: false,
```

Comment:
```suggestion
      disableSwitch: true,
```

This should be true, as our theme doesn't work with dark mode.